apps/src/StudioApp.js

```diff
@@ -2731,7 +2726,9 @@ StudioApp.prototype.setPageConstants = function (config, appSpecificConstants) {
     isEmbedView: !!config.embed,
     isShareView: !!config.share,
     pinWorkspaceToBottom: !!config.pinWorkspaceToBottom,
-    instructionsMarkdown: level.markdownInstructions,
+    shortInstructions: level.instructions,
+    // TODO - better handle the case where we have only short
+    instructionsMarkdown: level.markdownInstructions || level.instructions,
```

Comment:
Right now if we have just short instructions (true of some course1 levels). We'll end up using these as long instructions too, and our less/more toggle will essentially just hide the resizer. Long term we'll probably just want to show without the resizer/toggle button in this case.